### PR TITLE
Add smart folders

### PR DESCRIPTION
**Pull request by nxvl**

### CODE REVIEW

**Comment by maciejgryka** [comment]
@nxvl Did you try running a smart folder from our suite?

**Comment by nxvl** [comment]
Not yet.

File: lib/rainforest/cli/runner.rb [hunk 1]
```diff
@@ -92,6 +92,8 @@ def make_create_run_options
           # Not using git_trigger, so look for the
           if !options.tags.empty?
             post_opts[:tags] = options.tags
+          elsif !options.folder.nil?
+            post_opts[:smart_folder_id] = @options.folder.to_i
```
Comment (maciejgryka) [inline lib/rainforest/cli/runner.rb]: So the back-end already supports this and we only need to send more post data?

Comment (nxvl) [inline lib/rainforest/cli/runner.rb]: yup, that's how we start runs on smart folders from the app.

**Comment by nxvl** [comment]
@maciejgryka started a run succesfully!

**Comment by epaulet** [comment]
:shipit:

**Pull request merged**